views/js/pciCreator/dev/mathEntryInteraction/creator/widget/states/Question.js

```diff
@@ -13,20 +13,27 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2016 (original work) Open Assessment Technologies SA;
+ * Copyright (c) 2016-2017 (original work) Open Assessment Technologies SA;
  */
 
 define([
+    'jquery',
+    'i18n',
     'taoQtiItem/qtiCreator/widgets/states/factory',
     'taoQtiItem/qtiCreator/widgets/interactions/states/Question',
     'taoQtiItem/qtiCreator/widgets/helpers/formElement',
     'taoQtiItem/qtiCreator/editor/simpleContentEditableElement',
     'taoQtiItem/qtiCreator/editor/containerEditor',
     'tpl!mathEntryInteraction/creator/tpl/propertiesForm'
-], function(stateFactory, Question, formElement, simpleEditor, containerEditor, formTpl){
+], function($, __, stateFactory, Question, formElement, simpleEditor, containerEditor, formTpl){
     'use strict';
 
-    var MathEntryInteractionStateQuestion = stateFactory.extend(Question, function(){
+    var $addGapBtn = $('<div>', {
```

Comment:
that probably would be better in a template, next to the _propertiesForm_ for example